docs-src/steps.md

```diff
@@ -465,6 +465,27 @@ the email has been sent and contains:
 
 This will look for an email with 2 attachments.
 
+###`I store the email content matched by ":regex" as ":variableName" variable`
+
+Checks if there is an email on the configured mailing service and stores text body matched by regex inside variableStore.
+
+This steps takes first email from inbox, in order to find an email you're looking for clear inbox before sending email.
```

Comment:
I'm sure it should be `step`, not `steps`... anyways I would rewrite this sentence :P
```This step takes the first email from the inbox, in order to find an email you're looking for. It clears inbox before sending the email.```